### PR TITLE
Fix sentry module

### PR DESCRIPTION
**Pull request by sevanspowell**
Fixes up a couple of issues with my previous PR:
  - Updates versions of django, djangorestframework, and pyyaml to fix dependency vulnerabilities Dependabot reported (https://github.com/input-output-hk/ops-lib/pull/34 https://github.com/input-output-hk/ops-lib/pull/33 https://github.com/input-output-hk/ops-lib/pull/32) 
  - Updates the rust compiler from 1.41 to 1.42 to fix the symbolicator build (symbolicator is used by Sentry).

### CODE REVIEW

File: overlays/rust.nix [hunk 1]
```diff
@@ -0,0 +1,12 @@
+self: super:
+let
+  rustChannel = self.rustChannelOf { date = "2020-02-04"; channel = "nightly"; };
```
Comment (rvl) [inline overlays/rust.nix:3]: Do they have stable releases, or do you just pick a nightly to use?

File: pkgs/sentry/default.nix [hunk 2]
```diff
@@ -2,6 +2,12 @@
 
 let
   python = import ./requirements.nix { inherit pkgs; };
+
+  django_1_11_patch = pkgs.fetchpatch {
```
Comment (rvl) [inline pkgs/sentry/default.nix:6]: This is bringing back fond memories of Django - not!

File: overlays/rust.nix (hunk 1 above)
Comment (sevanspowell) [reply overlays/rust.nix:3]: I just picked the earliest nightly I could. Naersk uses an unstable flag that is only available in nightly rust builds.

File: pkgs/sentry/default.nix (hunk 2 above)
Comment (sevanspowell) [reply pkgs/sentry/default.nix:6]: Save me

**Pull request merged**